src/main/java/com/minecolonies/coremod/colony/requestsystem/management/manager/wrapped/WrappedBlacklistAssignmentRequestManager.java

```diff
@@ -31,7 +32,7 @@ public WrappedBlacklistAssignmentRequestManager(@NotNull final IStandardRequestM
      */
     @NotNull
     @Override
-    public void assignRequest(@NotNull final IToken token) throws IllegalArgumentException
+    public void assignRequest(@NotNull final IToken<?> token) throws IllegalArgumentException
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'java.lang.IllegalArgumentException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)